group-generators/helpers/data-providers/ens-subdomain/index.ts

```diff
@@ -0,0 +1,94 @@
+import { gql } from "graphql-request";
+import { EnsSubdomainsResponse, EnsDomainParams } from "./types";
+import { GraphQLProvider } from "@group-generators/helpers/data-providers/graphql";
+
+import { FetchedData } from "topics/group";
+
+export class EnsSubdomainProvider extends GraphQLProvider {
+  constructor() {
+    super({
+      url: `https://gateway.thegraph.com/api/${EnsSubdomainProvider.getAPIKey()}/subgraphs/id/EjtE3sBkYYAwr45BASiFp8cSZEvd1VHTzzYFvJwQUuJx`,
```

Comment:
The URL you used require an API key.

Why not using the one we cureently use for the ENS Resolver on the sismo hub (here: https://github.com/sismo-core/sismo-hub/blob/main/src/topics/resolver/ens-resolver.ts). This one: https://api.thegraph.com/subgraphs/name/ensdomains/ens/

With this request: 
```
query {
  domains(where: {name: "sismo.eth"}){
    subdomains {
      name
      owner {
        id
      }
    }
  }
}
```

Just in case, do you know the difference between the https://gateway.thegraph.com & https://api.thegraph.com/ URLS? I think it's related to the hosted and decentralized subgraph, but I'm ont well informed on this. (if you don't know no worries ahah)

Reply:
If I understand correctly, the hosted service (urls with api.thegraph.com) is somewhat discontinuing / migrating to the decentralized version 
https://thegraph.academy/developers/sunsetting-the-hosted-service/